src/index.ts

```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2016 Google Inc. All Rights Reserved.
+ * Copyright 2024 Google Inc. All Rights Reserved.
```

Comment:
nit: we normally do not update the copyright year of an existing file

```suggestion
 * Copyright 2016 Google Inc. All Rights Reserved.
```